imesec-full-stack.yml

```diff
@@ -83,6 +83,21 @@ services:
       placement:
         constraints: [node.role == manager]
 
+  bind:
+    container_name: bind
+    image: sameersbn/bind:9.11.3-20190706

```

Comment:
I think we don't want to be out on the latest updates of the image. Maybe change it to `latest`?